dockstore-webservice/src/main/java/io/dockstore/webservice/core/SourceFile.java

```diff
@@ -263,6 +263,23 @@ public Timestamp getDbUpdateDate() {
         return dbUpdateDate;
     }
 
+    /**
+     * Determine if the specified path matches the sourcefile path.
+     * The paths will match if, after adding a leading slash when it
+     * is missing, the resulting strings are equal, case insensitive.
+     * Thus, an absolute path and an absolute path missing the leading
+     * slash will match, as will paths that are the same string,
+     * whether absolute or relative.
+     */
+    @JsonIgnore
+    public boolean isSamePath(String otherPath) {
+        return otherPath != null && toAbsolutePath(getPath()).equalsIgnoreCase(toAbsolutePath(otherPath));
```

Comment:
Hmm, I'm wondering if we should be ignoring case. It's an edge case that may never get hit, but still GitHub does respect case...

Reply:
Well, we were ignoring case before. 
Also there are bizarre bugs with git on Mac and Windows when there are duplicates with case, so thinking we just leave it for platform cross-compatibility. 
https://www.hanselman.com/blog/git-is-casesensitive-and-your-filesystem-may-not-be-weird-folder-merging-on-windows

Bonus points'/ticket for a validation / message that warns that bad  things are afoot if there are duplicates when ignoring case?

Reply:
Link ticket I suppose?